src/discussions/post-comments/PostCommentsView.test.jsx

```diff
@@ -687,13 +693,131 @@ describe('ThreadView', () => {
+        .not
+        .toBeInTheDocument();
+    });
+
+    it('should have only two options', async () => {
+      renderComponent(discussionPostId);
+
+      await waitFor(() => screen.findByTestId('comment-comment-1'));
+      await act(async () => {
+        fireEvent.click(screen.getByRole('button', { name: /Oldest first/i }));
+      });
+      const dropdown = await waitFor(() => screen.findByTestId('comment-sort-dropdown-modal-popup'));
+
+      expect(dropdown)
+        .toBeInTheDocument();
+      expect(await within(dropdown)
+        .getAllByRole('button'))
+        .toHaveLength(2);
+    });
+
+    it('should be selected Oldest first and auto focus', async () => {
+      renderComponent(discussionPostId);
+
+      await waitFor(() => screen.findByTestId('comment-comment-1'));
+      await act(async () => {
+        fireEvent.click(screen.getByRole('button', { name: /Oldest first/i }));
+      });
+      const dropdown = await waitFor(() => screen.findByTestId('comment-sort-dropdown-modal-popup'));
+
+      expect(dropdown)
+        .toBeInTheDocument();
+      expect(within(dropdown)
+        .getByRole('button', { name: /Oldest first/i }))
+        .toBeInTheDocument();
+      expect(within(dropdown)
+        .getByRole('button', { name: /Oldest first/i }))
+        .toHaveFocus();
+      expect(within(dropdown)
+        .getByRole('button', { name: /Newest first/i }))
+        .not
+        .toHaveFocus();
+    });
+
+    test('successfully handles sort state update', async () => {
+      renderComponent(discussionPostId);
+
+      expect(store.getState().comments.sortOrder)
+        .toBeFalsy();
+
+      await waitFor(() => screen.findByTestId('comment-comment-1'));
+      await act(async () => {
+        fireEvent.click(screen.getByRole('button', { name: /Oldest first/i }));
+      });
+      const dropdown = await waitFor(() => screen.findByTestId('comment-sort-dropdown-modal-popup'));
+      await act(async () => {
+        fireEvent.click(within(dropdown)
+          .getByRole('button', { name: /Newest first/i }));
+      });
+
+      expect(store.getState().comments.sortOrder)
+        .toBeTruthy();
+    });
+    test('successfully handles tour state update', async () => {
+      const tourData = [{
+        id: 15,
+        tourName: 'not_responded_filter',
+        showTour: false,
+        enabled: true,
+        user: 8,
+      }, {
+        id: 16,
+        tourName: 'response_sort',
+        showTour: false,
+        enabled: true,
+        user: 8,
+      }];
+      await store.dispatch(fetchUserDiscussionsToursSuccess(tourData));
+      renderComponent(discussionPostId);
+      await waitFor(() => screen.findByTestId('comment-comment-1'));
+      expect(selectTours(store.getState()).find(item => item.tourName === 'response_sort').enabled)
+        .toBeTruthy();
+      await unmount();
+      expect(selectTours(store.getState()).find(item => item.tourName === 'response_sort').enabled)
+        .toBeFalsy();
```

Comment:
try to make it oneliner code. which is easy to read and manage as compared to multiple lines.  e.g `expect(selectTours(store.getState()).find(item => item.tourName === 'response_sort').enabled).toBeFalsy();`

add a new line between different stages of a test case. like
Initializations/rendering
New line
processing/actions
New line
expect/result

This pattern makes a test case more elegant and readable.